js/views/pageNavVw.js

```diff
@@ -591,14 +611,175 @@ module.exports = baseVw.extend({
       Backbone.history.navigate(route, {trigger: true});
     });
   },
+  
```

Comment:
The ideal would be to abstract this into an autosuggest view that can be re-used with any source of suggestions, so we could use it on other search fields.